### PR TITLE
yajl: new package at 2.1.0

### PR DESCRIPTION
**Pull request by alaughlin**
## Description of package
https://lloyd.github.io/yajl/

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libyajl/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+./configure \
+    -p /usr
```
Comment (alaughlin) [inline community/libyajl/build]: `--prefix` doesn't work for this package

**Comment by dylanaraps** [comment]
I would name this package `yajl`.

Also, `--prefix` looks to be supported: https://github.com/lloyd/yajl/blob/master/configure#L26

**Comment by alaughlin** [comment]
I can rename it.

I've tried `--prefix=/usr` and `--prefix /usr` and neither seemed to work 🤷‍♂

**Comment by dylanaraps** [comment]
OK, `-p` is fine then. :+1:

**Comment by alaughlin** [comment]
@dylanaraps renamed to `yajl` 👍

**Pull request merged**